### ISSUE TITLE
Install All Scripts warnings!

### ISSUE THREAD

**Issue by PawJershauge** [BrentOzarULTD/SQL-Server-First-Responder-Kit#3010]
**Version of the script**
Version:8.06
VersionDate = 20210914

Not actually a bug, but an incorrect order of execution within the hole script

**What is the current behavior?**
- The module **'sp_AllNightLog'** depends on the missing object **'master.dbo.DatabaseBackup'**. The module will still be created; however, it cannot run successfully until the object exists.
- The module **'sp_AllNightLog'** depends on the missing object **'master.dbo.sp_DatabaseRestore'**. The module will still be created; however, it cannot run successfully until the object exists.

**If the current behavior is a bug, please provide the steps to reproduce.**
Just run the **Install-All-Scripts.sql** script, and you will see the warnings.

**What is the expected behavior?**
No warnings

**Which versions of SQL Server and which OS are affected by this issue? Did this work in previous versions of our procedures?**
I has nothing to do with sql server version

**Comment by shermy** [comment]
i'm getting the same error when trying to install using Install-All-Scripts.sql

**Comment by BrentOzar** [comment]
Correct, when you're installing all of the scripts, there are dependencies on Ola Hallengren's DatabaseBackup stored procedure. If you don't want to install all scripts, try just installing the core ones.
